src/views/clusteroverview/OverviewTab/metric-charts-card/utils/hooks/types.ts

```diff
@@ -0,0 +1,5 @@
+export type ChartPoint = { x: Date; y: number; name?: string };
+
+export type ChartData = ChartPoint[];
+
+export type ChartDomain = { x: [left: number, right: number]; y: [top: number, bottom: number] };
```

Comment:
the y domain should be the other way around [bottom, top]

Reply:
Fixed!